Storage/src/Bucket.php

```diff
@@ -267,6 +267,18 @@ public function exists(array $options = [])
      *           Acceptable values include, `"authenticatedRead"`,
      *           `"bucketOwnerFullControl"`, `"bucketOwnerRead"`, `"private"`,
      *           `"projectPrivate"`, and `"publicRead"`.
+     *     @type array $retention Object retention configuration.
+     *     @type string $retention.retainUntilTime The earliest time in RFC 3339
+     *           UTC "Zulu" format that the object can be deleted or replaced.
+     *           This is the retention configuration set for this object.
+     *     @type string $retention.mode The mode of the retention configuration,
+     *           which can be either `"Unlocked"` or `"Locked"`.
+     *     @type string $retentionExpirationTime The earliest time in
```

Comment:
This parameter is not in the [Request Body](https://cloud.google.com/storage/docs/json_api/v1/objects/insert#request-body). We have confirmed that this parameter is correct, and most likely the public documentation needs to be updated.